dcrjson/dcrdextresults.go

```diff
@@ -5,6 +5,13 @@
 
 package dcrjson
 
+// GetStakeDifficultyResult models the data returned from the
+// getstakedifficulty command.
+type GetStakeDifficultyResult struct {
+	CurrentStakeDifficulty float64 `json:"currentstakediff"`
```

Comment:
Maybe just make these "Current" and "Next", the user should know they're a response to the original command